sync2/poller.go

```diff
@@ -741,3 +755,10 @@ func (p *poller) trackTimelineSize(size int, limited bool) {
 	}
 	p.timelineSizeVec.WithLabelValues(label).Observe(float64(size))
 }
+
+func (p *poller) trackGappyStateSize(size int) {
```

Comment:
You never call this?

Reply:
:facepalm: committed locally but not pushed.